curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/record-collection.english.md

```diff
@@ -30,20 +30,22 @@ You may refer back to <a href="learn/javascript-algorithms-and-data-structures/b
 
 ```yml
 tests:
+  - text: Do not modify the <code>collection</code> object
```

Comment:
I am not sure what verbiage should be here, but when I read this, it makes me think I need to return a copy of collection vs being able to modify the actual object. Maybe something like:
```suggestion
  - text: You should not change the original <code>collection</code> object's initialization.
```